moveit_setup_assistant/templates/moveit_config_pkg_template/launch/moveit_controller_manager.launch.xml

```diff
@@ -1,3 +1,6 @@
 <launch>
```

Comment:
Is modifying this file okay? like would it affect other already running stuff?

Reply:
add a comment explaining that the contents of this file can be replaced by other controller managers as desired, but that this is a good default for many integrations including Gazebo